source/documentation/troubleshooting/tracing.md

```diff
@@ -0,0 +1,9 @@
+## Tracing HTTP requests
+
+The Cloud Foundry router automatically adds Zipkin-compatible headers to the incoming HTTP requests so you can match those with your application's responses.
+
+> Zipkin is a tracing system that enables app developers to troubleshoot failures or latency issues. Zipkin provides the ability to trace requests and responses across distributed systems. See [Zipkin.io](https://zipkin.io) [external link] for more information.
+
+To trace app requests and responses in Cloud Foundry, apps must also log Zipkin headers.
```

Comment:
Do we need to elaborate on how they would do this?

Reply:
I don't think so, because it's about literally logging the Zipkin headers. This whole feature should be mostly for people who know what Zipkin is or they willing to learn it.